src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -112,6 +113,13 @@ public class SnowflakeSinkConnectorConfig {
   public static final String INGESTION_METHOD_DEFAULT_SNOWPIPE =
       IngestionMethodConfig.SNOWPIPE.toString();
 
+  // This is the streaming bdec file type which can be defined in config, default to Parquet
+  public static final String SNOWPIPE_STREAMING_FILE_TYPE = "snowflake.streaming.file.type";
+
+  // default file type to parquet
+  public static final String SNOWPIPE_STREAMING_FILE_TYPE_DEFAULT =
+      SnowpipeStreamingFileType.PARQUET.toString();
```

Comment:
Should we use write version numbers that we use internally for the Snowpipe Streaming Client SDK?

Reply:
Can we add a comment about this mapping? Or even a static enum map or something similar. I'm not entirely sure what the version numbers are or how they correspond to the connector

Reply:
I agree, i think using the file format name in the connector and adding a conversion enum as jay did below is more readable. Does add a little bit more dependency in testing for the conversion, but worth imo

Reply:
I agree with overall idea behind keeping same versions as of ingest SDK here. 
But here is my overall stance on why I like to decouple the behavior of ingest SDK and KC. 

1. Ingest SDK is just another dependency on SF Kafka Connector. Ingest SDK will evolve over time and we cant be tightly coupled with changes or changes coming into future. 
2. New file type in Ingest SDK would mean we will have a new version of it and KC will depend on it. The existing Unit and IT tests will help us catch any issues pertaining to the supported versions. 
3. Having to depend on ingest sdk would also mean, we will not have our own validators from KC side. 

Although, I also understand that both ingest sdk and KC is being developed by us so no harm is keeping it tightly coupled. 

Happy to make changes if folks like numbered versions here.

Reply:
I was not suggesting that we should keep the two tightly coupled, I was just trying to suggest we should make this a more generic change so when a new type is added (or KC needs to override some settings, for example, for Confluent Cloud, it may have certain memory constraint that we need to limit), there is no additional change required other than bumping the release version

Reply:
Ack, makes sense. Let me change to BDEC version directly but I am not entirely convinced about accepting json in this PR on in general overall.

Reply:
Strong agree with Toby, let's map to BDEC version directly.